### DISCUSSION TITLE
Problem when Cfloader my code

### DISCUSSION THREAD

**Discussion by Sheng-Qian-LI** [bitcraze/discussions#1195] [Q&A]
I want to Cfloader my code to aideck cache memory.But the terminal  stop at reset to bootloader mode as below.
Without flash img to memory from 0% to 100%.
How can I do? thanks.

<img width="463" alt="螢幕擷取畫面 2024-02-26 152441" src="https://github.com/bitcraze/discussions/assets/126442586/c7d9afb7-9e4c-40e8-8d61-b999af7a404d">

**Comment by gemenerik** [comment]
Can you flash any of the [AI deck examples](https://github.com/bitcraze/aideck-gap8-examples), e.g. [the hello world one](https://www.bitcraze.io/documentation/repository/aideck-gap8-examples/master/simple-examples/hello-world/)?

**Reply by Sheng-Qian-LI** [reply]
I have flashed examples of AI deck examples, but now I cannot flash other img to aideck.

1. Hello world example
![螢幕擷取畫面 2024-02-15 093221](https://github.com/bitcraze/discussions/assets/126442586/0accc9b9-c8cf-40bb-b01c-a4474e2a9c48)

2.My code(print image information from camera)
![螢幕擷取畫面 2024-02-23 223954](https://github.com/bitcraze/discussions/assets/126442586/d3714e7c-1257-445a-bf3c-03c1a2de2a26)

The above are successfully loaded onto AIDECK's memory using cfloader, but when I attempt to flash the modified code, it fails to flash.
This issue occurred suddenly; there were no problems with flashing before.

**Reply by knmcguire** [reply]
Just to clarify, have you updated anything else in the mean time like the crazyflie python library and client? Are you still able in your current state of your PC flash the original wifi example?

It is important to see if it an issue the python library or with the binary itself.

**Reply by Sheng-Qian-LI** [reply]
After being unable to flash, I attempted to use the cfclient's bootloader to update the firmware to the latest version. However, during the update, the Aideck section also fails to refresh.

**Reply by gemenerik** [reply]
For clarity, you didn't flash anything with JTAG debugger? This overwrites the OTA bootloader.

Otherwise, this sounds like a bug.

If you have access to a JTAG debugger, you can [reflash the bootloader](https://www.bitcraze.io/documentation/tutorials/getting-started-with-aideck/#gap8-bootloader) and see if this fixes the problem.
